src/components/map/Legend.tsx

```diff
@@ -9,40 +9,50 @@ import { MapSymbology } from "components/map/MapConfig"
-    dispatch({ type: 'TOGGLE_ESTIMATE_PINS' })
-  }
+  const handleLegendToggle = (e: React.MouseEvent<HTMLElement>) => {
+    const layerName: string = e.currentTarget.id;
+    switch (layerName) {
+      case "National": {
+        return dispatch({ type: "TOGGLE_NATIONAL_PIN_LAYER" });
+      }
+      case "Regional": {
+        return dispatch({ type: "TOGGLE_REGIONAL_PIN_LAYER" });
+      }
+      case "Local": {
+        return dispatch({ type: "TOGGLE_LOCAL_PIN_LAYER" });
+      }
+    }
+  };
 
   return (
     <div className={isMobileDeviceOrTablet ? "info flex legend-mobile center-item" : "info flex legend center-item"}>
       <div className="flex legend-container" key={Math.random()}>
-        <div className="legend-item cursor" onClick={clickEstimatePinsCheckbox}>
-          <i className="circleBase"><input className="checkbox" type="checkbox" checked={state.showEstimatePins} /></i>
-          <p className={isMobileDeviceOrTablet ? "mobile-text px-2" : "px-1"}>{Translate("StudyPins")}</p>
-        </div>
-        <div className="legend-item">
+        <div className="legend-item" id="National" onClick={handleLegendToggle}>
           <i className="circleBase" style={{ background: MapSymbology.StudyFeature.National.Color }}></i>
-          <p className={isMobileDeviceOrTablet ? "mobile-text px-2" : "px-1"}>{Translate("NationalStudies")}</p>
+          <p>{Translate("NationalStudies")}</p>
+          <input className="checkbox" type="checkbox" checked={state.explore.legendLayers.National} readOnly/>
         </div>
-        <div className="legend-item">
+        <div className="legend-item" id="Regional" onClick={handleLegendToggle}>
           <i className="circleBase" style={{ background: MapSymbology.StudyFeature.Regional.Color }}></i>
-          <p className={isMobileDeviceOrTablet ? "mobile-text px-2" : "px-1"}>{Translate("RegionalStudies")}</p>
+          <p>{Translate("RegionalStudies")}</p>
+          <input className="checkbox" type="checkbox" checked={state.explore.legendLayers.Regional} readOnly/>
         </div>
-        <div className="legend-item">
+        <div className="legend-item" id="Local" onClick={handleLegendToggle}>
           <i className="circleBase" style={{ background: MapSymbology.StudyFeature.Local.Color }}></i>
-          <p className={isMobileDeviceOrTablet ? "mobile-text px-2" : "px-1"}>{Translate("LocalStudies")}</p>
+          <p>{Translate("LocalStudies")}</p>
+          <input className="checkbox" type="checkbox" checked={state.explore.legendLayers.Local} readOnly/>
         </div>
         <div className="legend-item">
           <i className="block" style={{ background: MapSymbology.CountryFeature.HasData.Color, outlineWidth: 1, outlineStyle: "solid" }}></i>
-          <p className={isMobileDeviceOrTablet ? "mobile-text px-2" : "px-1"}>{Translate("CountryEstimatesExist")}</p>
+          <p>{Translate("CountryEstimatesExist")}</p>

```

Comment:
if we're no longer applying conditional styling on mobile vs desktop, did we test to see how it looks on both screen sizes?

Reply:
Legend is compact enough where we don't need to change styling for mobile. This version takes less space and looks better than the prev mobile legend
> 
> 
> ![image](https://user-images.githubusercontent.com/22464147/113496413-90d88600-94b6-11eb-850f-4df633b5c591.png)
> 
> Updated Legend to be more compact & use proper semantic ui check boxes